com.civicrm.hrjobroles/js/src/job-roles/controllers/job-roles.controller.js

```diff
@@ -475,6 +476,19 @@ define([
       });
     }
 
+    /**
+     * Trigger opening of editor window for editing provided option type
```

Comment:
Please always describe the function like "what it does":

`Opens editor window for editing a provided option type`